samples/osgi-bundles/org.wso2.carbon.uuf.sample.simple-auth.bundle/README.md

```diff
@@ -0,0 +1,147 @@
+## Pluggable Session Management
+
+This bundle uses a custom implementation of the [SessionManager.java](../../../components/uuf-core/src/main/java/org/wso2/carbon/uuf/spi/auth/SessionManager.java).
+
+### How to plug in a custom session manager
+
+**Step 1:**
+
+Write your custom implementation of the session manager, implementing
+the [SessionManager.java](../../../components/uuf-core/src/main/java/org/wso2/carbon/uuf/spi/auth/SessionManager.java) 
+interface. The session manager implemented in this sample is as shown below:
+
+```java
+public class PersistentSessionManager implements SessionManager {
```

Comment:
I don't think copy-pasting the `PersistentSessionManager` class here in the readme is a good idea. What hapens if someone change the `PersistentSessionManager.java` file. Then this part will be obsolete and invalid.